cienv/github_actions.go

```diff
@@ -24,15 +24,22 @@ type GitHubEvent struct {
 	} `json:"head_commit"`
 }
 
+type GitHubRepo struct {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported type GitHubRepo should have comment or be unexported